onadata/libs/utils/email.py

```diff
@@ -43,3 +42,22 @@ def get_verification_email_data(email, username, verification_url, request):
         })
 
     return email_data
+
+
+def get_account_lockout_email_data(username, end=False):
+    """Generates both the email upon start and end of account lockout"""
+    message_path = 'account_lockout/lockout_start.txt'
+    subject_path = 'account_lockout/lockout_email_subject.txt'
+    if end:
+        message_path = 'account_lockout/lockout_end.txt'
+    ctx_dict = {
+        'username': username,
+        'lockout_time': settings.LOCKOUT_TIME / 60
```

Comment:
Access to a setting that does not have a default value in our default settings. Could we use `getattr()` instead?
```python

'lockout_time': getattr(settings, 'LOCKOUT_TIME', 3600)  / 60)
```